sporks.json

```diff
@@ -862,9 +862,55 @@
       }
     },
     "testnet": {
+      "testnet34": {
+        "id": 34,
+        "live": true,
+        "name": "Testnet34",
+        "sporkTime": "2022-04-04T16:00:00Z",
+        "rootHeight": "64904846",
+        "rootParentId": "0cbb13d9e7ed7092b5f20d0a20a79d1cae96fd796fb2bc569b27ce49cc97d97e",
+        "rootStateCommitment": "f2e77e16628543e285be8bac677db06e17b37f49e53a107af1e6bf99fbc17b30",
+        "gitCommitHash": "32bda868ae651cd792f7f78dee31911141c5da8e",
+        "stateArtefacts": {
+          "gcp": {
+            "rootCheckpointFile": "https://storage.googleapis.com/flow-genesis-bootstrap/testnet-34-execution/public-root-information/root.checkpoint",
+            "rootProtocolStateSnapshot": "https://storage.googleapis.com/flow-genesis-bootstrap/testnet-34/public-root-information/root-protocol-state-snapshot.json",
+            "rootProtocolStateSnapshotSignature": "https://storage.googleapis.com/flow-genesis-bootstrap/testnet-34/public-root-information/root-protocol-state-snapshot.json.asc",
+            "nodeInfo": "https://storage.googleapis.com/flow-genesis-bootstrap/testnet-34/public-root-information/node-infos.pub.json",
+            "executionStateBucket": "flow_public_devnet34_execution_state",
+            "protocolDBArchive": "https://storage.googleapis.com/flow-execution-archive/devnet34/protocol-db.tar.gz",
```

Comment:
where does this come from? The values for the following are not correct and need to be updated but I am not sure where they come from?

> protocolDBArchive
> protocolDBArchiveChecksum
> executionStateArchive
> executio

nStateArchive

Reply:
you can leave these empty. They are for buckets where we've uploaded the protocol and execution state for historic sporks. We may populate them in the future

Reply:
thanks!